drivers/project_pass_fail/app/models/project_pass_fail/project_pass_fail.rb

```diff
@@ -197,7 +197,7 @@ def timeliness_threshold
 
     private def hmis_projects
       query = GrdaWarehouse::Hud::Project.where(id: filter.effective_project_ids)
-      query = query.with_project_type(filter.project_type_ids) if filter.project_type_ids.any?
+      query = query.with_project_type(filter.project_type_ids) unless filter.uses_default_project_type_codes
```

Comment:
So this means if project types have been explicitly chosen use them?
Do we need to check that the default for the filter on the page has them set to `[]`?  Otherwise they would have to be de-selected for this to get skipped.

Reply:
I'm not sure I understand the question... `uses_default_project_type_codes` defaults to true, and is set to false if by `FilterBase.update` if they are passed in via the hash...

Reply:
this can be removed, but the controller `create` method needs
```
@filter.update(filter_params)
      @report = report_class.create(options: @filter.for_params, user_id: @filter.user_id)
```